modules/nw-metallb-advertise-ip-pools-from-node-subset.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/about-advertising-ipaddresspool.adoc
+
+:_content-type: PROCEDURE
+
+[id="nw-metallb-advertise-ip-pools-to-node-subset_{context}"]
```

Comment:
Can you update the API ? I think we have it somewhere

Reply:
Removed nodeSelector Tech Preview note from BGPAdvertisement table.